scss/modals.scss

```diff
@@ -0,0 +1,52 @@
+.modal-backdrop,
+.modal-backdrop.fade.in {
+  opacity: 0.8;
+}
+
+.modal-show {
+  display: block;
+}
+
+@mixin modal-colored($color, $text-color-header) {
+  .modal-header {
+    color: $text-color-header;
+    background-color: $color;
+  }
+
+  .modal-footer .btn-modal {
+    @include button-colored($color, darken($color, 10), $text-color-header);
+  }
+}
+
+.modal-success{
+  @include modal-colored($btn-success-bg, $btn-success-color);
+}
+
+.modal-warning{
+  @include modal-colored($btn-warning-bg, $btn-warning-color);
+}
+
+.modal-danger{
+  @include modal-colored($btn-danger-bg, $btn-danger-color);
+}
+
+.modal-header {
+  h1,
+  h2,
+  h3 {
+    margin: 0;
+  }
+
+  .close {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.